data/competencies.yml

```
@@ -428,6 +430,17 @@
   area: technical
   domain: null
 
+- id: 152af675-554a-4d0d-afb1-190fca3867b2
```

UUID is a duplicate. Generated a new one:
```suggestion
- id: fd3e5d38-add3-4394-bd53-666482e422f2
```